docs/topics/braintree.rst

```diff
@@ -131,12 +131,23 @@ solitude.
             }
         }
 
-    :>json string braintree token: id of the payment method in braintree.
+    :>json string braintree token: id of the payment method in Braintree.
     :>json string braintree created_at: created date and time.
     :>json string braintree updated_at: updated date and time.
 
     :status 201: payment method created.
 
+To cancel a payment method. This will delete the payment method in Braintree
+and is not reversible.
+
+.. http:post:: /braintree/paymethod/cancel/
```

Comment:
Why cancel? That's confusing to me because you'd cancel a subscription which is different but related. The user would want to delete a payment method. Maybe ``/braintree/paymethod/delete/`` or ``/braintree/paymethod/remove/`` ?